src/Queue/Connection/ConfigFactory.php

```diff
@@ -0,0 +1,93 @@
+<?php
+
+namespace VladimirYuldashev\LaravelQueueRabbitMQ\Queue\Connection;
+
+use Illuminate\Support\Arr;
+use PhpAmqpLib\Connection\AMQPConnectionConfig;
+
+final class ConfigFactory
```

Comment:
This package used to make extensive use of `final` a couple years back and moved away from it. 
I'd prefer staying away from marking things `final`